### PR TITLE
docs: add initial set of prop type descriptions

### PR DESCRIPTION
**Pull request by joshblack**
Add initial set of prop type descriptions for components

#### Changelog

**New**

**Changed**

* A lot of component props now have prop types

**Removed**

### CODE REVIEW

File: src/components/ComposedModal/ComposedModal.js [hunk 1]
```diff
@@ -174,38 +231,58 @@ export class ModalHeader extends Component {
   }
 }
 
-export class ModalBody extends Component {
```
Comment (asudoh) [inline src/components/ComposedModal/ComposedModal.js]: Seems that you are cleaning up stuffs a bit, but I couldn’t find where `<ModalBody>` is gone - Any pointers?

Comment (joshblack) [reply src/components/ComposedModal/ComposedModal.js]: Haha yeah, totally missed this 😅 adding back in!

**Comment by joshblack** [comment]
cc @lovemecomputer

File: src/components/FileUploader/FileUploader.js [hunk 2]
```diff
@@ -115,17 +172,22 @@ export class FileUploaderButton extends Component {
...
     status: PropTypes.oneOf(['edit', 'complete', 'uploading']),
-    tabIndex: PropTypes.number,
-    onKeyDown: PropTypes.func,
   };
 
   static defaultProps = {
     onKeyDown: () => {},
     status: 'uploading',
     style: {},
-    tabIndex: 0,
```
Comment (asudoh) [inline src/components/FileUploader/FileUploader.js]: This prop seems to be included in `…other` portion. Changing default prop here seems to affect there?

Comment (joshblack) [reply src/components/FileUploader/FileUploader.js]: Good call!

**Pull request merged**